nixos/modules/config/terminfo.nix

```
@@ -45,7 +44,8 @@ with lib;
       tmux
       wezterm
       yaft
-    ]));
+    ] ++
+    lib.optional ("aarch64-linux" != pkgs.system) contour));
```

```suggestion
    lib.optional (!contour.meta.broken) contour));
```

Generally i prefer to stick with a single source of truth

---

Perhaps `filter (drv: !drv.meta.broken) [...]` makes sense